src/follow/grpc_test.go

```diff
@@ -42,7 +42,9 @@ func TestClientToServerCommunication(t *testing.T) {
 	// Note that the ordering of things here is pretty important; we need to get the
 	// client connected & ready to receive events before we push them all into the
 	// server and shut it down again.
-	serverState := core.NewBuildState(5, nil, 4, core.DefaultConfiguration())
+	config := core.DefaultConfiguration()
+	config.Please.NumThreads = 5
```

Comment:
Necessary? Could you just use DefaultBuildState here?

Reply:
nope, this test is specifically testing that this value gets propagated so it needs to be something that is not (usually) the default.